.github/workflows/linux.yml

```diff
@@ -43,6 +44,14 @@ jobs:
           skip_when_only_listed_labels_set: 'docs'
           skip_when_only_listed_files_changed: '*.md,*.rst,*.png,*.jpg,*.svg'
 
+      - name: Get validation cycle to run
+        id: val_cycle
+        shell: bash
+        run: |
+          val_cycle="${{ (contains(github.event.pull_request.labels.*.name, 'nightly') || 
+                          github.event_name == 'schedule') && 'nightly' || 'precommit' }}"
+          echo "val_cycle=$val_cycle" >> "$GITHUB_OUTPUT"
```

Comment:
is it expected that we will have a single `nightly` or per test suite `tf-nightly` / `pytorch-nightly`, which can will be converted by smart-ci to `needs.smart_ci.outputs.pytorch_val_cycle`, etc ?

Reply:
I would like to be able to run pytorch separate from tf if that is possible.